docs/gettingstarted/byoe-rhel8.md

```diff
@@ -1,38 +1,36 @@
-# SC4S "Bring Your Own Environment"
+# Configure SC4S in a custom environment
```

Comment:
maybe something like `non-containerized SC4S deployment`? 
Because the only difference between this and other deployments is that here we do bare-metal deployment, and in others we use podman/docker containers

Reply:
something like this? "# Configure SC4S in a non-containerized SC4S deployment

Configuring SC4S in a non-containerized SC4S deployment requires a custom configuration. Consider this configuration only if:"

we probably still want to talk about what is and what isn't supported, for now i took out the "we don't support it" part, because it seems like we do, but to a limited extent. Any suggestions?

Reply:
The changes version sounds great, thank you. please only restore the part related to the support